python-sdk/src/astro/files/base.py

```diff
@@ -25,6 +25,8 @@ class File(LoggingMixin, Dataset):
     :param conn_id: Airflow connection ID
     :param filetype: constant to provide an explicit file type
     :param normalize_config: parameters in dict format of pandas json_normalize() function.
+    :param pd_kw: The Pandas related kwargs as a dict.
+        This will be supplied to Pandas ``read_csv`` function etc.
```

Comment:
It feels that both the (previously existing) `normalize_config` and the (just introduced) `pd_kw` could have more explicit names. In both scenarios, we want to give arguments to underlying Pandas functions - it may make sense to have a single keyword argument. What if we had something like:
```
{
    "pandas_kwargs":
        "json_normalize": 
        "read_csv": 
}
```